product_tax_multicompany_default/tests/test_product_tax_multicompany.py

```diff
@@ -17,27 +19,49 @@ def setUpClass(cls):
             {"name": "Test company 2", "country_id": default_country.id}
         )
         group_account_manager = cls.env.ref("account.group_account_manager")
+        group_account_user = cls.env.ref("account.group_account_user")
+        group_multi_company = cls.env.ref("base.group_multi_company")
         ResUsers = cls.env["res.users"]
         cls.user_1 = ResUsers.create(
```

Comment:
Please use the method `test_new_user` for simplifying code:

https://github.com/odoo/odoo/blob/08859ca7939454303e34dc39c831ac8f75773c3d/odoo/tests/common.py#L124

Example: https://github.com/odoo/odoo/blob/08859ca7939454303e34dc39c831ac8f75773c3d/addons/web/tests/test_ir_model.py#L37

Reply:
Done